test/test-lib/fixtures.ts

```diff
@@ -1,7 +1,148 @@
+import { sbvrUtils } from '@resin/pinejs';
 import * as Bluebird from 'bluebird';
+import * as fs from 'fs';
 import * as _ from 'lodash';
+import * as path from 'path';
+
+import { Tx } from '@resin/pinejs/out/database-layer/db';
+import { Headers } from 'request';
+import { API_HOST } from '../../src/lib/config';
+import { requestAsync } from '../../src/lib/request';
+
+const { api, root } = sbvrUtils;
+
+interface ApiRequestOpts {
+	url: string;
+	resource: string;
+	headers: Headers;
+	method: string;
+	json: boolean;
+	body: AnyObject;
+}
 
 type Fixtures = Dictionary<PromiseLike<Dictionary<PromiseLike<any>>>>;
+type SemiAwaitedFixtures = Dictionary<Dictionary<PromiseLike<any>>>;
```

Comment:
```suggestion
type PendingFixtures = Dictionary<Dictionary<PromiseLike<any>>>;
```

?

Reply:
I renamed this one to `PartiallyAppliedFixtures` and the one above to `PendingFixtures` (since they're slightly more accurate names)